node/coinstacks/solana/api/src/service.ts

```diff
@@ -0,0 +1,38 @@
+import type { Account, API, Tx, TxHistory } from './models'
+import { BaseAPI, handleError,  SendTxBody } from '@shapeshiftoss/common-api'
+import { Helius } from "helius-sdk";
+
+export interface ServiceArgs {
+  rpcUrl: string
+  rpcApiKey: string
+}
+
+export class Service implements Omit<BaseAPI, 'getInfo'>, API {
+  private readonly heliusSdk: Helius
+
+  constructor(args: ServiceArgs) {
+    this.heliusSdk = new Helius(args.rpcApiKey)
```

Comment:
we should pass rpc url in case we ever point to nownodes or something else

Reply:
Yeah this is kinda why I kept them in the args, but I couldn't instantiate them without adding an eslint disable comment, anyway it will be removed by the service removal naturally 👍

Reply:
Done in [412c348](https://github.com/shapeshift/unchained/pull/1049/commits/412c348ee52620a3028caadd8322dd58348bbd20)

Reply:
I wonder if we really needed this? 

I would completely separate NOWNodes instances from the Helius SDK and use web3js with NOWNodes

The Helius SDK is adding another parameter (the Helius API key) to the RPC URL which we don't need with NOWNodes + it adds in houses methods that we don't need with a regular RPC

Reply:
Yeah I guess that was the point I was trying to get at. To me it feels like we should use web3 for raw jsonrpc calls so that we may switch out rpc urls/api keys as needed, and helius sdk for actual helius api calls. But for now, happy to send things as is, not like it would be a hard update if/when we ever need to use nownodes + helius